package.json

```diff
@@ -44,20 +44,21 @@
   },
   "peerDependencies": {
     "react": "^16.13.1 || ^17.0.0",
-    "react-dom": "^16.13.1 || ^17.0.0"
+    "react-dom": "^16.13.1 || ^17.0.0",
```

Comment:
The upgrade helper uses `^17.0.2` now, should we switch to that as well?

Reply:
Seems the dependency is `^17.0.2`, but the defined peerDependency is `^17.0.0`. Probably a mismatch on their side. It should be fine to the `17.0.2` here, will test it first and then commit it

Reply:
Just updated to the version. Nothing broke so far